helm-charts/k8s-mediaserver/jackett/templates/jackett-resources.yml

```diff
@@ -1,12 +1,10 @@
-{{ if .Values.jackett.enabled }}
----
 ### CONFIGMAPS
 ## INIT-CONTAINER
 apiVersion: v1
 data:
   ServerConfig.json: |
     {
-      "BasePathOverride": "{{ .Values.jackett.ingress.path }}"
```

Comment:
Each chart being self contained means qualified paths like this one don't need the `.jackett.` portion anymore.